equipment_handler/source/tempsense_athena.c

```diff
@@ -23,25 +23,23 @@
 #define CHANNEL_LOCAL 0
 #define CHANNEL_REMOTE 1
 
-uint8_t tmp_addr[6] = {0x1D, 0x1E, 0x2A, 0x4C, 0x4F, 0x4E};
+uint8_t tmp_addr[6] = {0x49, 0x4A};
 
-void inittemp_all(void) {
+void inittemp_all(void){
     int i;
     int delay;
-    for (i = 0; i < 6; i++) {
+    for(i=0;i<2;i++){
         tmp421_init_client(tmp_addr[i]);
-        for (delay = 0; delay < 0x1000; delay++)
-            ; // temporary fix... don't want delay down the road
+        for (delay = 0; delay < 0x1000; delay++);//temporary fix... don't want delay down the road
```

Comment:
Since we've started only running code like this in task contexts, you should add a vTaskDelay

Reply:
if it's a temporary fix and we don't want delay down the road, can we just leave it and delete when it's not needed anymore. Not sure if revising it will be worth the time

Reply:
@Taylorr82 what we should really have is a driver here that is immune to timing (that's the only way to get rid of this 'temporary' fix). until then making a task delay is preferred.

Reply:
i have replaced these lines with vtaskdelays, it should be okay to put them in the same location right? I don't think they need to be called inside the infinite for(::) ?